### DISCUSSION TITLE
input  type=“number” value={ value }

### DISCUSSION THREAD

**Discussion by marcelloh** [a-h/templ#689] [Ideas]
when I pass an int  as a value to an input  type=“number”, I expect not to do value={ fmt.Sprintf("%d", value) } but just to use it like this value={ value } or do I miss some explanation?

**Comment by nicolasfranck** [comment]
Everything within `{}` should be a string. gohtml of course would accept this of course, but templ wants to generate go-code (in which that is also not accepted).
